apps/web-giddh/src/app/vouchers/create/create.component.html

```diff
@@ -1437,61 +1437,73 @@ <h2 class="d-inline-block">
+                                                                                        <span>{{ localeData?.adjust_amount
+                                                                                            }}</span>
+                                                                                        <span class="text-gray">
+                                                                                            <amount-field [amount]="
+                                                                                                    adjustPaymentData.totalAdjustedAmount
+                                                                                                        | replace
+                                                                                                            : '-'
+                                                                                                            : ''
+                                                                                                " [currencySymbol]="
+                                                                                                    isMulticurrencyAccount
+                                                                                                        ? account?.baseCurrencySymbol
+                                                                                                        : false
+                                                                                                " [currencyCode]="
+                                                                                                    isMulticurrencyAccount
+                                                                                                        ? account?.baseCurrency
+                                                                                                        : false
+                                                                                                "></amount-field>
+                                                                                        </span>
+                                                                                        <i (click)="openAdjustmentDialog(true)"
+                                                                                            class="icon-edit text-blue font-12"></i>
+                                                                                    </div>
+                                                                                        <i class="icon-trash font-16 cursor-pointer mr-l1" *ngIf="invoiceType.isSalesInvoice"
+                                                                                        (click)="deleteDepositRow(depositIndex)"></i>
```

Comment:
confirm with kriti for show delete button on sales invoice only.

Reply:
is confirmed with Kriti???

Reply:
YES , sales Invoice and  Bill